src/components/Loading Screen/LoadingScreen.js

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components/macro'
+import BounceLoader from 'react-spinners/BounceLoader'
+
+export default function LoadingScreen() {
+  return (
+    <Wrapper>
+      <BounceLoader color={'var(--darker-grey)'} />
```

Comment:
I am not a fan of adding CSS to JSX (HTML), instead, I prefer separating structure and styling.
Maybe it's possible to change the color in CSS/StyledComponents?

Reply:
That is a good point and maybe i will change my style later. But at the moment i am used to mix CSS sometimes with JSX. I did it in some other occasions as well.